draft-ietf-taps-interface.md

```diff
@@ -1840,7 +1835,7 @@ Default:
 : false
 
 This property specifies that a message should be sent and received as a single
-packet without transport-layer segmentation or network-layer fragmentation.
+packet without transport-layer segmentation or network-layer fragmentation, if possible.
```

Comment:
So, it's okay for this to fail? Or for the singular send to succeed, but for a middlebox to re-segment the data? The receiver therefore cannot rely on this property even if it knows a priori that the sender employed it.

Are we being clear enough that negotiated properties do not comprise a contract between sender and receiver?

Reply:
Yes, it's ok for this to fail. Regarding the receiver and negotation: this is all one-sided!  There isn't any negotiation here. A TAPS system can talk to a TAPS system on the other side, but it can also talk to a "normal" TCP peer, for instance.